### ISSUE TITLE
Would it be possible to add the NRLiSt BDB dataset? [moleculenet]

### ISSUE THREAD

**Issue by UnixJunkie** [deepchem/deepchem#1577]
Hello,

This one is a dataset for nuclear receptors.
Only experimentally tested molecules.
For each target, molecules are separated into agonists and antagonists.
Very high quality dataset, hand curated by a pharmacist.

---
Lagarde, N., Ben Nasr, N., Jérémie, A., Guillemain, H., Laville, V., Labib, T., ... & Montes, M. (2014). NRLiSt BDB, the manually curated nuclear receptors ligands and structures benchmarking database. Journal of medicinal chemistry, 57(7), 3117-3125.
---

Dataset summary:
Protein target | # Agonists | # Antagonists
AR− | 179 | 179
ERα- | 74 | 434
ERα+ | 102 | 132
ERβ+ | 70 | 70
GR− | 204 | 295
GR+ | 74 | 369
PR− | 269 | 269
PR+ | 74 | 531
RARα− | 41 | 133
RXRα− | 114 | 210

For this one also, I should have a standardized version somewhere.
I could turn it into a bunch of CSV files.

Regards,
F.

**Comment by UnixJunkie** [comment]
Am I in the wrong git repository?

**Comment by UnixJunkie** [comment]
@rbharath I am waiting for some feedback in order to push this one further.

**Comment by rbharath** [comment]
Yes, for sure, this would be welcome! If you turn it into CSV files, I can upload onto the DeepChem S3 bucket.
